apps/core/scenes/search/PlacePicker/PlacePickerContent.js

```diff
@@ -158,12 +166,24 @@ class PlacePickerContent extends React.Component<Props, State> {
           onChangeText={this.handleChangeText}
           options={filterOptions(options, selectedOptions)}
           selected={selectedOptions}
+          onKeyPress={this.handleBackspacePress}
```

Comment:
`onKeyPress` sounds like this would trigger on any key press.  So when I press `a`, `onKeyPress` does nothing but somehow the text changes.  I think the logic becomes confusing with `onChangeText` too. Could we improve the API?

Reply:
OptionPicker's onKeyPress callback trigger when you press whatever key, all logic for filtering keys is in the handleBackspacePress.

Reply:
I'm quite ok with current solution. But maybe we want to somehow extend `onKeyPress` later and for example add `enter` to confirm/close input or arrows to jump between places (if I want to for example delete just first place from three).

So what about adding something like `handleKeyPress`. And handle different key cases inside (for now just backspace)?

Reply:
@JoseDuSV sure, why not?  I will change it as you suggest.

Reply:
done